trello/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,37 @@
+= Trello Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Trello (Trello Connector) enables you to integrate custom applications into Trello to facilitate faster collaboration between teams.
+
+For information about compatibility and fixed issues, see the Trello Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Trello API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to:
+
+* The Trello target resource
+* Anypoint Platform
+* Anypoint Studio version 4.2.2 or later
```

Comment:
4.2.2 is the Mule runtime engine version not the version of Anypoint Studio. Could devs confirm if this connector can be used with Studio 7.8?